packages/expo-gl-cpp/cpp/EXGLContext.cpp

```diff
@@ -154,7 +154,9 @@ void EXGLContext::maybeReadAndCacheSupportedExtensions() {
         if (extensionName.substr(0, 3) == "GL_") {
           extensionName.erase(0, 3);
         }
-        supportedExtensions.insert(extensionName);
+        if (extensionName != "OES_vertex_array_object") {
+          supportedExtensions.insert(extensionName);
```

Comment:
Implementation is intentionally minimal to avoid conflicts with in progress refactor that will land after the next sdk release